packages/grafana-data/src/field/fieldDisplay.ts

```diff
@@ -64,6 +64,7 @@ export interface FieldDisplay {
   rowIndex?: number; // only filled in when the value is from a row (ie, not a reduction)
   getLinks?: () => LinkModel[];
   hasLinks: boolean;
+  fieldType?: FieldType;
```

Comment:
is this just kicking the can... can we just add the field so we can use it later

Reply:
maybe `source` or `sourceField`?

Reply:
Good idea. I'll do that.